src/index.js

```diff
@@ -245,22 +245,28 @@ export default class EmojiReact extends Component {
 	}
 
 	render() {
-		const { reactions, onReaction, onEmojiClick } = this.props;
+		let { reactions, onReaction, onEmojiClick, sort } = this.props;
+		if (sort) {
+		reactions = reactions.slice(0);
+	    reactions.sort(function (a, b) {
```

Comment:
does this work? sort does not alter array in place ,so without assigning this to a var it seems as if it would do nothing